lib/engine/game/g_2038/ability/launch.rb

```diff
@@ -0,0 +1,16 @@
+# frozen_string_literal: true
+
+require_relative '../../../ability/base'
+
+module Engine
+  module Ability
+    class Launch < Base
+      attr_reader :style, :corporation
```

Comment:
doesn't seem like style is being used? are there other abilities that could do this?

Reply:
Style is not yet used.  But there are several kinds of launching and I expect to handle them from this class.

There are currently no abilities explicitly for this activity.  It would probably be possible to abuse existing activities in some way to accomplish this functionality but it would make the code less clear because anyone reading the code would be expecting something else than what will actually happen.